Makefile.work

```diff
@@ -366,6 +367,11 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            KUBERNETES_VERSION=$(KUBERNETES_VERSION) \
                            KUBERNETES_CNI_VERSION=$(KUBERNETES_CNI_VERSION) \
                            K8s_GCR_IO_PAUSE_VERSION=$(K8s_GCR_IO_PAUSE_VERSION) \
+                           INCLUDE_KUBERNETES_MASTER=$(INCLUDE_KUBERNETES_MASTER) \
+                           MASTER_KUBERNETES_VERSION=$(MASTER_KUBERNETES_VERSION) \
+                           MASTER_PAUSE_VERSION=$(MASTER_PAUSE_VERSION) \
+                           MASTER_COREDNS_VERSION=$(MASTER_COREDNS_VERSION) \
+                           MASTER_ETCD_VERSION=$(MASTER_ETCD_VERSION) \
```

Comment:
We should not pass the K8s detail package version info by the sonic slave command line.
You can add a config file in rules, for instance, rules/k8s.mk, the make file will be loaded into slave.mk automatically.

Reply:
Changed